readgroup.wdl

```diff
@@ -66,39 +64,34 @@ workflow Readgroup {
         String chunksR2 = "${readgroupDir}/chunk_${chunk}/${chunk}_2.fq.gz"
     }
 
-    call biopet.FastqSplitter as fastqsplitterR1 {
-        input:
-            inputFastq = copyR1.outputFile,
-            outputPaths = chunksR1,
-            dockerTag = dockerTags["biopet-fastqsplitter"]
-    }
-
-
-    if (defined(readgroup.reads.R2)){
-        call biopet.FastqSplitter as fastqsplitterR2 {
+    if (numberChunks > 1) {
+        call biopet.FastqSplitter as fastqsplitterR1 {
             input:
-                inputFastq = select_first([copyR2.outputFile]),
-                outputPaths = chunksR2,
+                inputFastq = copyR1.outputFile,
+                outputPaths = chunksR1,
                 dockerTag = dockerTags["biopet-fastqsplitter"]
         }
+        if (defined(readgroup.reads.R2)){
```

Comment:
Do `if` statements work inside of `if` statements nowadays?

Reply:
Yes. They always have worked.